core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -800,6 +800,11 @@ class ReplicaManager(val config: KafkaConfig,
       }
 
       logManager.truncateTo(partitionsToMakeFollower.map(partition => (new TopicAndPartition(partition), partition.getOrCreateReplica().highWatermark.messageOffset)).toMap)
+      partitionsToMakeFollower.foreach(partition => {
```

Comment:
Related to @auradkar's point, do we need to update the comment in `makeFollowers`?